node/cli/src/benchmarking.rs

```diff
@@ -127,7 +126,7 @@ pub fn create_benchmark_extrinsic(
 		)),
 		common_runtime::extensions::check_nonce::CheckNonce::<runtime::Runtime>::from(nonce),
 		frame_system::CheckWeight::<runtime::Runtime>::new(),
-		pallet_transaction_payment::ChargeTransactionPayment::<runtime::Runtime>::from(0),
+		pallet_frequency_tx_payment::ChargeFrqTransactionPayment::<runtime::Runtime>::from(0),
```

Comment:
Don't forget to re-run overhead benchmark if we're changing the SignedExtensions